src/visualizers/Turtle.ts

```diff
@@ -388,7 +411,7 @@ case you inadvertently left out a value.)
                     + '`M` - Maximum index, '
                     + '`A(...)` -- sequence entry at any index, '
                     + '`b` - current bearing, `x`,`y` - current position, '
-                    + '`f` - frame number (triggers animation).'
+                    + '`f` - frame number (allows changing path shape).'
```

Comment:
I'm not sure this phrase "allows changing path shape" is clear.  Do we need something here at all?  Maybe "frame number" is clear enough.